internal/am/upload_transfer.go

```diff
@@ -57,7 +51,8 @@ func (a *UploadTransferActivity) Execute(
 	ctx context.Context,
 	params *UploadTransferActivityParams,
 ) (*UploadTransferActivityResult, error) {
-	a.logger.V(1).Info("Execute UploadTransferActivity", "SourcePath", params.SourcePath)
+	logger := temporal_tools.GetLogger(ctx)
+	logger.V(1).Info("Execute UploadTransferActivity", "SourcePath", params.SourcePath)
```

Comment:
A thought for the future: I've always seen this logging entry as unnecessary becuse the SDK produces a similar event automatically if I remember correctly, it does not log local variables or parameters but inputs are available in the Temporal workflow history anyways.

Reply:
I said the same in https://github.com/artefactual-sdps/temporal-activities/pull/3#discussion_r1566247913. But @djjuhasz likes it.

Reply:
@sevein I find the log entry helpful for debugging failures where Temporal schedules the activity (so it shows up in the workflow history) but the worker doesn't actually run the `Execute()` method.  Echoing the parameters is probably unnecessary in that case though.

Reply:
Perhaps that can be done from the worker interceptor too (ActivityInboundInterceptor), I will investigate.